web-client/src/presenter/actions/CaseDeadline/getCaseDeadlinesAction.ts

```diff
@@ -1,12 +1,5 @@
 import { state } from 'cerebral';
 
-/**
- * get case deadlines between start and end date
```

Comment:
I'm on board with removing jsdoc but I hate to see human-readable descriptions go away, especially when the function name is far less clear than the human-readable description was.

Reply:
added back the jsdoc and added to it